_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -52,6 +52,7 @@ class HealthCareApp extends React.Component {
     this.context.store.dispatch(ensureFieldsInitialized(path));
     if (validations.isValidSection(path, sectionData)) {
       hashHistory.push(this.getUrl('next'));
+      sectionData.sectionComplete = true;
```

Comment:
This is a change from Courtnye's PR #1758, and shouldn't be included in this PR.